### ISSUE TITLE
Having trouble getting started

### ISSUE THREAD

**Issue by xird** [audiocogs/aurora.js#149]
I thought I'd try out Aurora, but I'm getting stuck right away. I started by building aurora.js, and then copy-pasting the code from the Getting Started article to an empty HTML page:

```
<script>
var player = AV.Player.fromFile("test.wav");
</script>
```

"Uncaught ReferenceError: AV is not defined"

Oh, right, need to include the file.

```
<script src="aurora.js"></script>
<script>
var player = AV.Player.fromURL('http://mysite.com/audio.wav');
</script>
```

"Uncaught ReferenceError: require is not defined(anonymous function)
Uncaught ReferenceError: AV is not defined"

Huh. I guess you need to use RequireJS.

```
<script src="require.js"></script>
<script>
require(['aurora'], function () {
  var player = AV.Player.fromFile("test.wav");
});
</script>
```

"Uncaught Error: Module name "src/aurora" has not been loaded yet for context: _. Use require([])
http://requirejs.org/docs/errors.html#notloadedE
ReferenceError: AV is not defined(…)"

I thought I was doing that?

Any pointers on what I'm doing wrong?

**Comment by crackofdusk** [comment]
Hey @xird,

The way I usually go about this is to use the pre-built files (look at the releases pages for various codecs, e.g.: https://github.com/audiocogs/aurora.js/releases)

Here is an example I just tried:

```html
<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>Aurora test</title>
  <script src="js/aurora.js"></script>
  <script src="js/mp3.js"></script>
  <script>
    var player = AV.Player.fromURL('test.mp3');
  </script>
</head>
<body>
</body>
</html>
```

I ran ` python -m http.server 8080` in the working directory to avoid problems with local file access. Then I visited http://localhost:8080 and in the browser console I entered `player.play()`.

That should get you going.
